sdk/identity/azure-identity/src/main/java/com/azure/identity/CredentialBuilderBase.java

```diff
@@ -68,4 +68,17 @@ public T httpPipeline(HttpPipeline httpPipeline) {
         this.identityClientOptions.setHttpPipeline(httpPipeline);
         return (T) this;
     }
+
+    /**
+     * Sets the duration before the actual expiry of a token to refresh it.
+     * This is useful when network is congested and a request containing the
+     * token takes longer than normal to get to the server.
```

Comment:
You should document what the default value is if it is not set.

Reply:
Documented.